lib/guardian.rb

```diff
@@ -353,8 +353,6 @@ def can_see_invite_emails?(user)
   def can_invite_to_forum?(groups = nil)
     authenticated? &&
     (SiteSetting.max_invites_per_day.to_i > 0 || is_staff?) &&
-    !SiteSetting.enable_discourse_connect &&
-    SiteSetting.enable_local_logins &&
```

Comment:
This needs to check for `SiteSetting.enable_local_logins && Discourse.enabled_auth_providers.count > 0`. However... it seems unlikely that people will shoot themselves in the foot. Do we even want to care about this case?

Reply:
Sounds good :) Will not try and handle the crazy edge case then.